Tools/autotest/vehicle_test_suite.py

```diff
@@ -4372,7 +4372,7 @@ def TestLogDownloadMAVProxyNetwork(self, upload_logs=False):
         self.context_push()
         self.set_parameters({
             "NET_ENABLED": 1,
-            "LOG_DISARMED": 1,
+            "LOG_DISARMED": 0,
```

Comment:
drive by? this may make for much longer test times

Reply:
Yep, this was stuff left in here which was broken out into a separate PR.

Removed it